code/__DEFINES/misc_defines.dm

```diff
@@ -104,6 +104,9 @@
 #define STAGE_FIVE 9
 #define STAGE_SIX 11 //From supermatter shard
 
+/// A define for the center of the coordinate map of big machinery
+#define SRC 0
```

Comment:
This is way too general of a name for a define that goes across the whole codebase, I suggest something like this
```suggestion
#define BIG_MACH_CENTER 0
```

Reply:
Very good point!